source/channels/find-channels.rst

```diff
@@ -7,16 +7,30 @@ Find channels
 Find channels to join
```

Comment:
Propose renaming this to "Browse channels" so it isn't getting mixed up with the modal titles in the navigate between channels page